src/Host/Client/Impl/Extensions/RStringExtensions.cs

```diff
@@ -103,6 +103,14 @@ public static class RStringExtensions {
             return s.Replace("/", "\\");
         }
 
+        public static string ProjectRelativePathToRemoteProjectPath(this string path, string remoteRoot, string projectName) {
+            if (string.IsNullOrWhiteSpace(projectName)) {
+                return ($"{remoteRoot}/{path}")?.ToRPath();
```

Comment:
No.

---
In reply to: [80395220](https://github.com/Microsoft/RTVS/pull/2405#discussion_r80395220) [](ancestors = 80395220)